extensions/BMO/template/en/default/global/choose-product.html.tmpl

```diff
@@ -98,14 +98,6 @@
       icon="component.png"
     %]
     [% INCLUDE easyproduct
-      name="Firefox OS"
-      icon="firefox_os.png"
-    %]
-    [% INCLUDE easyproduct
-      name="Marketplace"
-      icon="marketplace.png"
-    %]
-    [% INCLUDE easyproduct
       name="Thunderbird"
```

Comment:
It might be time to remove Thunderbird from this list, given our current position on it.